### ISSUE TITLE
Last `<p>` is missing in a list

### ISSUE THREAD

**Issue by Massedil** [erusev/parsedown#667]
Try to parse this markdown code [here](http://parsedown.org/demo) : 

```
> 1. bla
> 
> 2. bla
> 
> 3. bla
```

Markdown PHP 1.3 gives : 

```
<blockquote>
  <ol>
  <li><p>bla</p></li>
  <li><p>bla</p></li>
  <li><p>bla</p></li>
  </ol>
</blockquote>
```

Parsedown gives : 

```
<blockquote>
<ol>
<li>
<p>bla</p>
</li>
<li>
<p>bla</p>
</li>
<li>bla</li>
</ol>
</blockquote>
```

The last `<p></p>` is missing.

The result with parsedown is `<li>bla</li>` instead of `<li><p>bla</p></li>`.

---

Problem is also here with : 

```
1. bla

2. bla

3. bla
```

Same result with : 

```
1. bla
2. bla
3. bla
```

**Comment by aidantwoods** [comment]
Hi, which version of Parsedown are you using? I believe this should have been fixed in the latest release (1.7.1) by #475

**Comment by Massedil** [comment]
> Hi, which version of Parsedown are you using? I believe this should have been fixed in the latest release (1.7.1) by #475

Sorry, I was not using version 1.7.1. I can confirm that this issue is solved in 1.7.1.

Note that the demo website (http://parsedown.org/demo) seems to be not up to date.

**Comment by aidantwoods** [comment]
> Note that the demo website (http://parsedown.org/demo) seems to be not up to date.

Yeah :( I don't have access to update that unfortunately
